src/main/scripts/test-in-container

```diff
@@ -9,5 +9,5 @@ echo 'Testing tutorial in user environment...'
 make -j`nproc`
 cd ~
 echo "module TEST imports BOOL endmodule" > test.k
-kompile test.k --backend ocaml
+kompile test.k --backend llvm
```

Comment:
The reason we are building with the ocaml backend here is because we have already tested the other backends. Changing this means we will not get informed if something relating to the ocaml backend breaks, but it doesn't actually add any new test coverage. Please undo this change.

Reply:
Done @dwightguth